tutorials/abap-s4hanacloud-procurement-purchasereq-checks/abap-s4hanacloud-procurement-purchasereq-checks.md

```diff
@@ -1,31 +1,35 @@
 ---
-title: Implement a Business Add-in (`BAdI`) To Check a Purchase Requisition
-description: Define and implement a developer extension (`BAdI`), which performs checks during creation of a purchase requisition item.
+parser: v2
 auto_validation: true
 time: 30
-tags: [ tutorial>intermediate, software-product>sap-btp--abap-environment, software-product-function>s-4hana-cloud-abap-environment]
-primary_tag: programming-tool>abap-development
+tags: [ tutorial>intermediate, programming-tool>abap-development]
+primary_tag: software-product-function>s-4hana-cloud-abap-environment
 author_name: Julie Plummer
 author_profile: https://github.com/julieplummer20
 
 ---
 
+# Implement a Business Add-in (`BAdI`) To Check a Purchase Requisition
+<!-- description --> Define and implement a developer extension (`BAdI`), which performs checks during creation of a purchase requisition item.
+
 ## Prerequisites
-- **IMPORTANT**: It is essential that you are a member of SAP Early Adopter program
 - You have a license for SAP S/4HANA Cloud and have a developer user in this system
 - You have installed [SAP ABAP Development Tools (ADT), latest version](https://tools.hana.ondemand.com/#abap), and have created an ABAP Cloud project for your SAP S/4HANA Cloud System in it
 - You are familiar with the concept of extensions to the SAP standard and with `BAdIs` in particular. If not, see the **More Information** section at the end of this tutorial
 - To test the `BAdI`: In SAP Fiori launchpad, you have the authorization for the app **Process Purchase Requisitions - Professional**, i.e. **`SAP_BR_PURCHASER`**
 
 
-## Details
-### You will learn
+## You will learn
 - How to log on to SAP S/4HANA Cloud ABAP Environment
 - How to create an ABAP package
 - How to find relevant existing `BAdI` enhancement spots for your line of business, in this case **Materials-Management-Purchasing**
-- How to implement checks for the field quantity and delivery date during creation of purchase requisition item
+- How to implement a `BAdI` that checks for the field quantity and delivery date during creation of purchase requisition item
+
+
+
+## Intro
+>**NOTE**: The administrator receives an welcome e-mail after provisioning. This e-mail includes the system URL. By removing `/ui` you can log into the SAP S/4HANA Cloud ABAP Environment system. Further information can be found [Developer Extensibility: Connect to the ABAP System](https://help.sap.com/docs/SAP_S4HANA_CLOUD/6aa39f1ac05441e5a23f484f31e477e7/4b962c243a3342189f8af460cc444883.html?locale=en-US).
```

Comment:
Please do not start a note with the label "NOTE".
Please check the style guide for details: https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21